src/test/java/io/kestra/plugin/microsoft365/outlook/GetTest.java

```diff
@@ -0,0 +1,70 @@
+
+package io.kestra.plugin.microsoft365.outlook;
+
+import com.microsoft.graph.models.ItemBody;
+import com.microsoft.graph.models.Message;
+import com.microsoft.graph.serviceclient.GraphServiceClient;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.runners.RunContextFactory;
+import io.kestra.plugin.microsoft365.outlook.utils.GraphMailUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.ArgumentMatchers.any;
+
+import io.kestra.core.junit.annotations.KestraTest;
+import jakarta.inject.Inject;
+
+@KestraTest
+public class GetTest {
+
+    @Mock
+    private GraphServiceClient graphServiceClient;
+
+    @Inject
+    private RunContextFactory runContextFactory;
+
+    private Get get;
+
+    @BeforeEach
+    void setUp() {
+        get = Get.builder()
+            .id("get-task")
```

Comment:
For each tasks in tests please add something like this to avoid flakiness:

```suggestion
            .id("get-task-" + IdUtils.create())
```